datacake-eventual-consistency/src/test_utils.rs

```diff
@@ -565,3 +565,6 @@ impl Storage for MemStore {
         Ok(docs.into_iter())
     }
 }
+
+unsafe impl Send for MemStore {}
+unsafe impl Sync for MemStore {}
```

Comment:
Just noticed this, but we can't do this as A) we dont have a justification for why it's safe and B) this should be auto-implemented and can be removed.

Reply:
Ops, my bad. It's an experiment that got committed! :(